src/sentry/replays/usecases/query/__init__.py

```diff
@@ -450,15 +451,22 @@ def _select_from_fields() -> list[Column | Function]:
 
 
 def execute_query(query: Query, tenant_id: dict[str, int], referrer: str) -> Mapping[str, Any]:
-    return raw_snql_query(
-        Request(
-            dataset="replays",
-            app_id="replay-backend-web",
-            query=query,
-            tenant_ids=tenant_id,
-        ),
-        referrer,
-    )
+    try:
+        return raw_snql_query(
+            Request(
+                dataset="replays",
+                app_id="replay-backend-web",
+                query=query,
+                tenant_ids=tenant_id,
+            ),
+            referrer,
+        )
+    except RateLimitExceeded as exc:
+        sentry_sdk.set_tag("replay-rate-limit-exceeded", True)
+        sentry_sdk.set_extra("org_id", tenant_id.get("organization_id"))
```

Comment:
we don't have `org_id` already instrumented at a different level, so all events have it set?

Also: should it be a tag instead?

```suggestion
        sentry_sdk.set_tag("org_id", tenant_id.get("organization_id"))
```